runtime-sdk/modules/evm/src/lib.rs

```diff
@@ -80,43 +109,30 @@ pub enum Error {
     #[sdk_error(code = 2)]
     EVMError(String),
 
-    #[error("invalid signer type")]
-    #[sdk_error(code = 3)]
-    InvalidSignerType,
```

Comment:
can you explain the removal of this? will the evm module now accept calls from all signers?

I do see that derive_caller supports other signer types. but the sdk account derivation wouldn't match between (i) ed25519 signature -> oasis-staking-compatible account and (ii) ed25519 signature -> H160 caller -> ethereum-compatible sdk account

Reply:
Oh I see, maybe we need to fix this, hm.

Reply:
Yeah so while this works it would result in the address mapping into a different SDK account. You could still control the funds and everything would work, except that SDK account could never be used to sign any transactions, it would just be used to store the EVM balance. We could forbid these to make it less confusing for users.

Reply:
Yeah I think forbidding it would be best, it can always be allowed later if we decide this would be fine after all.